api-tests/go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/Percona-Lab/pmm-api-tests
```

Comment:
Do we need separate modules configuration for tests?

Reply:
Yes, I think it's better to keep own go module for each "micro-service". API tests are standalone. We don't need dependencies from API tests inside pmm-managed.